react-sdk.code-workspace

```diff
@@ -2,11 +2,14 @@
 	"folders": [
 		{
 			"path": "."
+		},
+		{
+			"path": "../../scripts"
```

Comment:
I don't think we can assume that everyone has a scripts directory 2 levels up. (I know I don't). And I think this will cause problems. So, I'd revert this change with another commit to put it back the way it was. Then, this PR looks ok.

Reply:
ok, tried original commit from VSCode.  Staged more than just the readme.md

Reply:
Thanks!